fedimint-client/src/sm/executor.rs

```diff
@@ -292,8 +280,10 @@ impl Executor {
             self.inner
                 .log_event_dbtx(
                     dbtx,
-                    StateMachineCreated {
+                    StateMachineUpdated {
                         operation_id,
+                        started: true,
+                        terminal: false,
                         module_id: state.module_instance_id(),
```

Comment:
Sad we don't have a way to reference SMs, but nothing to do about it now I guess.